tests/test_common/test_system.py

```diff
@@ -1,37 +1,16 @@
 # -*- coding: utf-8 -*-
 import hashlib
-import os
-import pathlib
 import io
+import pathlib
+from os.path import abspath, join
 
```

Comment:
It is possible (and possibly recommended?) to use pathlib methods only?

Reply:
That should be possible yes :) I'll fix that.

Reply:
Now uses pathlib only.